central/deployment/datastore/datastore_test_constructors.go

```diff
@@ -44,6 +47,25 @@ func NewTestDataStore(
 	}
 	deploymentStore := pgStore.FullStoreWrap(pgStore.New(testDB.DB))
 	searcher := search.NewV2(deploymentStore)
+	mockCtrl := gomock.NewController(t)
```

Comment:
Same as before, avoid mocking datastores inside GetTestPostgresDataStore functions as callers usually expect these to be real datastores.